test/globalSetup.ts

```diff
@@ -11,13 +11,13 @@ export default async function setup() {
 	// @ts-expect-error - this is how we pass data to tests
 	globalThis.FIXTURES = await Promise.all(
 		glob(`${fixRoot}/*.json`).map((f) =>
-			compile(f).then((policyDocument) => {
+			compile(f).then((policyDocuments) => {
```

Comment:
changing this to `compile(f).then(([policyDocument])` would also save you a `[0]` down below on line 20.